### PR TITLE
"open" action now have consistent format with "write"

### PR DESCRIPTION
**Pull request by Alaya-in-Matrix**

### CODE REVIEW

File: plugin/activity-log.vim [hunk 1]
```diff
@@ -159,25 +174,25 @@ endfunction
...
-	let l:branch = ''
-
-	if g:activity_log_append_git_branch
-		let l:branch = system('cd ' . expand("%:h") . "; git branch --no-color 2> /dev/null | sed -e '/^[^*]/d'")
-		if (l:branch =~ "^* ")
-			let l:branch = substitute(l:branch, '\* ', '', '')
-			let l:branch = substitute(l:branch, '\n', '', '')
-		endif
-	endif
+" function TimeLogLine(action)
```
Comment (AD7six) [inline plugin/activity-log.vim]: These two functions need consolidating, rather than just deleting one of them.

File: plugin/activity-log.vim [hunk 2]
```diff
@@ -61,10 +61,10 @@ augroup END
...
 "" sets ,ls mapping to log current line as start of task in .log files
 "" sets ,le mapping to log current line as end of task in .log files
-nmap <silent> <Leader>ls :call TimeLogLine('start_task')<CR>
-nmap <silent> <Leader>le :call TimeLogLine('end_task')<CR>
-nmap <silent> <Leader>li :call TimeLogLine('interval_task')<CR>
-nmap <silent> <Leader>lc :call TimeLogLine('continue_task')<CR>
+" nmap <silent> <Leader>ls :call TimeLogLine('start_task')<CR>
+" nmap <silent> <Leader>le :call TimeLogLine('end_task')<CR>
+" nmap <silent> <Leader>li :call TimeLogLine('interval_task')<CR>
+" nmap <silent> <Leader>lc :call TimeLogLine('continue_task')<CR>
```
Comment (AD7six) [inline plugin/activity-log.vim]: :+1: That should never have been there =) - the equivalent should still work though, and be mentioned in the readme though.

File: plugin/activity-log.vim (hunk 1 above)
Comment (Alaya-in-Matrix) [inline plugin/activity-log.vim]: It is the commit "82c2fa4" that my merge request was intended for, the current commit that commented `TimeLogLine` out was committed accidentally...perhaps I need a revert

Comment (Alaya-in-Matrix) [inline plugin/activity-log.vim]: It seems that as long as the pull request is still open, any new commit to forked project would be added into the pull request?

Comment (AD7six) [inline plugin/activity-log.vim]: If you only want that one commit:

    git reset --hard 82c2fa4
    git push -f

**Comment by AD7six** [comment]
Am not sure why this wasn't merged many moons ago - but I'll be archiving this project as I don't use or maintain it these days :).

**Pull request closed without merging**